src/sidebar/activity/tree/ActivityTreeDataProvider.ts

```diff
@@ -20,8 +23,9 @@ class ActivityTreeDataProvider extends TreeDataProvider {
     }
 
     public async getRoots(): Promise<TreeItem[]> {
-        // TODO: Handle errors
-        const { data } = await ActivityStore.get();
+        const { data, errors } = await ActivityStore.get();
+
+        const roots: TreeItem[] = errors?.length ? [new ActivityErrorsTreeItem(errors)] : [];
```

Comment:
I think we should move this below activities calculation where we push into it.

Reply:
Error is the first item we show, so I calculated it first